csrc/device_lower/pass/index.cpp

```diff
@@ -1872,8 +1872,8 @@ Val* IndexLowering::getIterationIndexForBroadcast(
 
 void IndexLowering::handle(const PadOp* pad) {
   // Convert to a where op as:
-  // consumer[consumer_idx] = (producer_idx >= 0 && producer_idx <
-  //                           producer_extent) ?
+  // consumer[consumer_idx] = (consumer_idx >= left_pad && consumer_idx <
```

Comment:
Don't remember why I originally used the producer logical indices. The override index isn't necessary if the consumer indices are used.